.env.in

```diff
@@ -1,7 +1,2 @@
-MONGODB_URI=[Connection String from Azure, how to change it read in Readme, https://github.com/KTH/kurs-pm-data-api/blob/master/README.md#prepara-database-in-azure]
-API_KEYS_0=?name=kursinfo-web&apiKey=[generate a password for public pages]&scope=read
-API_KEYS_1=?name=kurs-pm-data-admin-web&apiKey=[generate a password for public pages]&scope=write&scope=read
-APPLICATIONINSIGHTS_CONNECTION_STRING=[Azure, Application insights, connection string, can be found in Overview]
-USE_COSMOS_DB=true
+MONGODB_URI=[Available in Azure KeyVault]
 LOGGING_ACCESS_LOG=debug
```

Comment:
Could you check in serverSettings.js if `useAccessLog` only is set to true if `LOGGING_ACCESS_LOG` is set to `true`? If yes, we can remove `LOGGING_ACCESS_LOG` from .env.in as well, because it does not have any effect.